### DISCUSSION TITLE
Multiple GPU Support

### DISCUSSION THREAD

**Discussion by ThioJoe** [ggml-org/llama.cpp#1657] [Ideas]
I know that supporting GPUs in the first place was quite a feat. And I think an awesome future step would be to support multiple GPUs. 

I'm sure many people have their old GPUs either still in their rig or lying around, and those GPUs could now have new purpose for accelerating the outputs. And especially for those who may specifically go out and buy more GPUs to be able to increase the speed of the largest models.

**Comment by Green-Sky** [comment]
see https://github.com/ggerganov/llama.cpp/pull/1607

**Reply by ThioJoe** [reply]
Ah very cool

**Comment by unnir** [comment]
see https://github.com/ggml-org/llama.cpp/pull/1703
